### DISCUSSION TITLE
添加代理服务器设置

### DISCUSSION THREAD

**Discussion by ANGJustinl** [immersive-translate/immersive-translate#2353] [Ideas]
### 新功能 | New Feature

1.添加独立的 **代理服务器设置** 如在进阶设置中添加`proxy`配置，允许用户将请求经过本地代理。

### 为什么您想要添加这些功能 | Why you want to add these features

1.同样**使用OpenAI SDK接口**的很多服务提供商**禁止大陆IP调用api服务** (如 chatGPT，Grok，Cluade等)，允许用户在进阶设置中使用自己本地代理，可以允许用户**自主解决**此问题。

### 提议 | Proposal

设置代理后，所有请求经由用户本地代理通过

**Comment by function2-llx** [comment]
可以通过直接对浏览器配置代理来解决吧

**Reply by ANGJustinl** [reply]
确实可以，但是需要全局代理，如proxy-omega等规则代理则无作用。

**Reply by function2-llx** [reply]
没有用过你提到的这个工具，不过只要你的代理客户端支持自定义规则的话，应该不是什么问题

**Reply by ANGJustinl** [reply]
ok谢谢，不过还是希望有办可以单独设置一个插件内代理XD

**Comment by ANGJustinl** [comment]
可以通过只配置自己设定的 `自定义 API 接口地址 -> proxy` 来经过代理，已经解决。
